test/files/42-http_0_9-method_only.t

```diff
@@ -1,3 +1,2 @@
 >>>
-HELLO

```

Comment:
why was this test replaced? It seems like a very different thing to test?

Reply:
HTTP 0.9 has only one method and it is `GET`

Maybe we can rename the test instead of changing its content

Reply:
Yeah lets leave the old test in place as well. It should be handled in some way as well right? Or was the handling wrong?

Reply:
Ok, I am renaming the old test, and adding a new with real HTTP 0.9